cluster-essentials/test/clusterinit_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
```

Comment:
Test file should be in the same directory as the file being tested. No need for a new package.